jwst/associations/tests/helpers.py

```diff
@@ -63,35 +56,42 @@ class BasePoolRule():
     valid_rules: list = []
 
     def test_rules_exist(self):
+        """Test that level 3 registry rules are valid."""
         rules = registry_level3_only()
-        assert len(rules) >= len(self.valid_rules)
+        assert len(rules) >= len(self.valid_rules)  # noqa: S101
```

Comment:
It might be good to add a global rule for this.  Right now we ignore all ruff rules in files that start with `test`; this file is exempt because it's a helper, and I think that's a good thing.  But it makes sense for test helpers to have asserts in them like this.  So maybe we need an additional rule that anything inside a `tests/` directory can ignore certain rules like this one?

Reply:
I considered this - in the end I figured that it'd be nice to have some rules applied to his file, and I thought about excluding just this rule. I could do that.

Reply:
I will leave it up to you.  It looks like it'd be a one-liner because we already have some docs rules ignored for everything inside a `tests/` folder: https://github.com/spacetelescope/jwst/blob/6cb593526baa0f83b6ec62fcba1321b00b6cd5b9/.ruff.toml#L89